code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -35,7 +35,7 @@
 	damage = 10
 	stamina = 30
 	armour_penetration = -30
-	ricochets_max = 6
+	ricochets_max = 1
```

Comment:
Undocumented change

Reply:
the projectile does say that its supposed to be bouncy

Reply:
I reduced it to 1 bounce because the damage is neglectable (reduced to 80% every time it hits a wall) and may end up bouncing off 2 walls and going rampant or hitting the detective back.

Reply:
they're rubber, sounds funny